src/config/syscheck-config.c

```diff
@@ -1664,10 +1664,10 @@ int Read_Syscheck(const OS_XML *xml, XML_NODE node, void *configp, __attribute__
     }
```

Comment:
There are still many `merror` in this file, have you checked them if they are correct?

Reply:
I check only those associated with the configurations that were in the table.

Reply:
Done